### DISCUSSION TITLE
How to model a transparent material with measured BRDF?

### DISCUSSION THREAD

**Discussion by sebastienfricker** [mitsuba-renderer/mitsuba3#1012] [Q&A] [answered]
I'm very new to Mitsuba but so far I love the tool! My goal is to model a transparent glass with a coating. I know the BRDF of the coating, so I'm thinking of using the "measured material" bsdf to model it. But if I understand correctly, in this model it's assumed that all light that's not reflected gets absorbed. Instead, I would like to transmit it through the material. How can I achieve that?
Thanks in advance.

**Comment by njroussel** [comment, marked as answer]
Hi @sebastienfricker 

You are correct, that model does not account for any form of transmission.

The naive way is just to to use the `blendbsdf` with some other transmissive material.

However I would imagine that you'd want the transmissive lobe to somewhat proportional to the incident angle, assuming some Fresnel-like behavior. For that I think you're better off writing your own custom BSDF. You can obviously still keep a nested inner `measured` BSDF for the reflective component.

**Reply by sebastienfricker** [reply]
Hi @njroussel ,
thank you very much for pointing me in the right direction.
I had missed the fact that it's possible to write a custom BSDF! I'll try this option, making a modified version of the dielectric BSDF where the reflectance depends on the coating properties.
